### PR TITLE
fix

### PR DESCRIPTION
**Pull request by georgeDaskalakis**
@PavelBal can you please test the behaviour with the dataset you had? Now the view stays enabled, but you cannot select datasets anymore (the last one remains plotted) and the import button becomes disabled. Also the error is displayed in both tabs.

### CODE REVIEW

**Comment by georgeDaskalakis** [comment]
@msevestre you can check the behaviour in the Starter of Core - since it does not change anything about the behaviour after the data is imported. I fyou need any help on this just call me :)

File: src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs [hunk 1]
```diff
@@ -26,6 +28,21 @@ public string SelectedSeparator
          get => separatorComboBoxEdit.SelectedText;
       }
 
+      public void DisableSelectingDataSets(string errorMessage)
```
Comment (msevestre) [inline src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs:31]: I don't like this. You are mixing two properties and that's why you are duplicating the call (look the first 3 lines are the same in the Enable and Disable)

You may need to methods, but it should be " bool SelectingDataSetsEnable {set}" (potentially get but you probably don't need it aside from tests)

And then you need a method "SetErrorMessage(string message)"

File: src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs [hunk 2]
```diff
@@ -104,10 +104,10 @@ private void plotDataSet(object sender, DataSetSelectedEventArgs e)
          }
          catch (InvalidArgumentException invalidException)
          {
-            _view.DisableConfirmationView();
             var errors = new ParseErrors();
             errors.Add(_dataSource.DataSetAt(e.Index), new NonMonotonicalTimeParseErrorDescription(Error.ErrorWhenPlottingDataRepository(e.Index, invalidException.Message)));
```
Comment (msevestre) [inline src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs:108]: I know it does not come from this commit...but WHY is there this explicit `NonMonotonicalTimeParseErrorDescription` coming out of nowhere??

**Review by msevestre — CHANGES_REQUESTED** [review]
ALso it feels that this "behavior" should be covered by tests at the presenter level at least

File: src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs (hunk 1 above)
Comment (georgeDaskalakis) [reply src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs:31]: @msevestre  I like your suggestion way better! But then you mean listen to the propertyChanged to actually set the properties? If at some point you get 5 min time let's discuss this on a call

Comment (msevestre) [reply src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs:31]: No. I'll hope on a call with you later today

Comment (georgeDaskalakis) [reply src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs:31]: @msevestre call anytime :)

Comment (msevestre) [reply src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs:31]: I am on slack.

**Comment by georgeDaskalakis** [comment]
@msevestre I have done the changes we discussed (also did add a getter for the test), and I think this should be ready now.

**Pull request merged**